vdom/src/com/vdom/core/IndirectPlayer.java

```diff
@@ -212,7 +212,13 @@ public int selectInt(MoveContext context, Card responsible, int maxInt, Object e
                 options[i + 1] = int_options[i];
             }
         }
-        return int_options[selectOption(context, responsible, options)];
+        int choice = selectOption(context, responsible, options);
```

Comment:
I disagree on this one. It's better to know that something went wrong in case another default choice would be needed for some reason.
But I can change it, it's your call.